Led.cpp

```diff
@@ -1,7 +1,52 @@
 #include "Led.h"
 
-Led::Led() {
+Led::Led() : PIN(0) {
 }
 
-Led::~Led() {
-}
\ No newline at end of file
+Led::Led(unsigned pin) : PIN(pin) {
+    //	pinMode(pin, OUTPUT);
+}
+
+void Led::setBrightness(unsigned percent) {
+    if (ledPower) {
+        brightness = percent * 2.55;
+    }
+    //	analogWrite(led, brightness);
```

Comment:
when I said "active brightness" I mean the brightness of the currently on led. If the led is on we need to change the active brightness. It it is on or off we still need to remember the brightness setting.